test/integration/local/processing/process-payment-requests-manual-ledger.test.js

```diff
@@ -0,0 +1,240 @@
+const db = require('../../../../app/data')
+const config = require('../../../../app/config')
+const processPaymentRequests = require('../../../../app/processing/process-payment-requests')
+const moment = require('moment')
+const { IRREGULAR } = require('../../../../app/debt-types')
+const mockSendMessage = jest.fn()
+jest.mock('ffc-messaging', () => {
+  return {
+    MessageSender: jest.fn().mockImplementation(() => {
+      return {
+        sendMessage: mockSendMessage,
+        closeConnection: jest.fn()
+      }
+    })
+  }
+})
+let paymentRequest
+let schedule
+let invoiceLine
+
+describe('process payment requests with manual ledger flag', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks()
+    await db.sequelize.truncate({ cascade: true })
+
+    await db.scheme.create({
+      schemeId: 1,
+      name: 'SFI',
+      active: true
+    })
+
+    await db.accountCode.create({
+      accountCodeId: 1,
+      schemeId: 1,
+      lineDescription: 'G00 - Gross value of claim',
+      accountCodeAP: 'SOS273',
+      accountCodeARIrr: 'SOS274',
+      accountCodeARAdm: 'SOS275'
+    })
+
+    await db.holdCategory.create({
+      holdCategoryId: 1,
+      schemeId: 1,
+      name: 'Manual ledger hold'
+    })
+
+    paymentRequest = {
+      paymentRequestId: 1,
+      schemeId: 1,
+      frn: 1234567890,
+      marketingYear: 2022,
+      invoiceNumber: 'S12345678SIP123456V001',
+      value: 100,
+      paymentRequestNumber: 1,
+      debtType: IRREGULAR
+    }
+
+    invoiceLine = {
+      invoiceLineId: 1,
+      paymentRequestId: 1,
+      description: 'G00 - Gross value of claim',
+      schemeCode: '80001',
+      fundCode: 'DRD10',
+      value: 100
+    }
+
+    schedule = {
+      scheduleId: 1,
+      paymentRequestId: 1,
+      planned: moment().subtract(1, 'day')
+    }
+  })
+
+  afterAll(async () => {
+    await db.sequelize.truncate({ cascade: true })
+    await db.sequelize.close()
+  })
+
+  test('should process manual ledger request and create hold if useManualLedgerCheck equals true when delta value is < 0', async () => {
```

Comment:
This should be glued to the bottom of `test/integration/local/processing/process-payment-requests.test.js`